acceptance-tests/features/steps/make_deed_effective_definitions.rb

```diff
@@ -0,0 +1,35 @@
+Given(/^I have made the created deed effective$/) do
+  sign_the_deed(@deed_id, 'SIGNATURE', '1')
+  make_deed_effective(@deed_id)
+end
+
+When(/^I get the deed from the api$/) do
+  @deed = get_deed_data(@deed_id)
+end
+
+Then(/^the signature is applied to the deed$/) do
+  assert(@deed['deed']['operative-deed']['registrars-signature'],
+         'Signature not applied')
+end
+
+Then(/^the date is applied to the deed$/) do
+  assert(@deed['deed']['operative-deed']['date-effective'],
+         'Date not applied')
+end
+
+Given(/^I have created a case$/) do
+  @case_id = create_case_data
+end
+
+Given(/^I have linked the created deed and case$/) do
+  update_case_deed
+end
+
```

Comment:
A minor thing really, but I would be tempted to put steps that could be used in other user stories further down the line in the general_definitions.rb (steps like ' I have created a case' and 'I have linked the created deed to the case')